src/EnergyPlus/General.cc

```diff
@@ -186,6 +186,7 @@ namespace General {
 
         X0 = X_0;
         X1 = X_1;
+        XTemp = X0;
```

Comment:
Can you explain why this XTemp assignment is needed?  I was expecting to see the `break` logic below, but I don't understand this change.  Is it fixing a different issue?

Reply:
@Myoldmopar The XTmep was assigned inside the While loop and is used as an output value. When an if section was added as below:
            if (std::abs(X1 - X0) < SMALL) {
                break;
            }
The While loop was out before XTemp was assigned. Since XTemp was not initialized before exit, I added an initial value for XTemp.

Reply:
Thanks!! That makes perfect sense! :+1: